jekyll/_cci2/overview.md

```diff
@@ -0,0 +1,128 @@
+
+## Production Installation Requirements
+
+- Minimum of two instances, one for the Services component and one for the Builders. Both instances require running a Linux distribution with Docker support, such as Ubuntu Trusty, or the CircleCI Amazon Machine Image (AMI). 
+- The Services instance must have at least eight CPUs and 32GB RAM. 
+- The Builders instance must have six CPUs or GPUs and 10GB RAM and a root volume of at least 50GB.
+- Both instances must have network access to a GitHub or Github Enterprise instance. 
+
+To determine the size required for your workload, provision two CPUs for processing plus two CPUs and 4GB of memory for each container. For example, six CPUs and 10GB of RAM supports two containers concurrently because the default container size is two CPUs with 4GB. 
+
+### Production Installation Options
+
+It is possible to install CircleCI 2.0 in either of the following configurations:
+
+- A single Services machine with one or more Builder machines. See [Installing CircleCI on Amazon Web Services Manually]({{ site.baseurl }}/enterprise/aws-manual/) or [Installing CircleCI on Amazon Web Services with Terraform]({{ site.baseurl }}/enterprise/aws/) for instructions.
+- A single Services machine with four database hosts for HA and one or more Builder machines, see [Installing CircleCI in a High Availability Configuration]({{ site.baseurl }}/enterprise/high-availability/) for instructions.
+
+## Customer Use Cases 
+
+**Coinbase** runs CircleCI behind their firewall for security and reliability. The [Coinbase case study report](https://circleci.com/customers/coinbase/) reveals that using CircleCI reduced their average time from merge to deploy in half, reduced operations time spent on continuous integration maintenance from 50% of one person's time to less than one hour per week, and increased developer throughput by 20%. 
+
+**Fanatics** increased their team efficiency by 300% with CircleCI. See the [Fanatics case study report](https://circleci.com/customers/fanatics/).
+
+CircleCI enables **Cruise Automation** (a subsidiary of GM) to run many more simulations before putting code in a road test, see the [Cruise case study report](https://circleci.com/customers/cruise/).
+
+Refer to the [CircleCI Customers page](https://circleci.com/customers/) for the complete set of case studies for companies large and small who are using CircleCI.
+
+## Basic Features
+
+After a software repository in GitHub or GitHub Enterprise is added as a project to the CircleCI application, every new commit triggers a build and notification of success or failure through webhooks with integrations for Slack, HipChat, Campfire, Flowdock, or IRC notifications. Code coverage results are available from the details page for any project for which a reporting library is added.
+
+CircleCI may also be configured to deploy code to various environments, including the following:
+
```

Comment:
This list is feeling a bit dated -- we should probably consider just saying that we can deploy to all major platforms via scripting, as the 2.0 path to deploy won't be through the existing UI-based components.

Reply:
ok, I think the list is helpful, but I made the text change to say 'all major platforms'